modal/sandbox.py

```diff
@@ -424,13 +430,24 @@ async def exec(
         ```
         """
 
+        if workdir is not None and not workdir.startswith("/"):
+            raise InvalidError(f"workdir must be an absolute path, got: {workdir}")
+
+        # Force secret resolution so we can pass the secret IDs to the backend.
+        for secret in secrets:
```

Comment:
I'm looking for advice here on how to hydrate these secrets. Overwriting `_hydrate_lazily` feels very wrong, but I couldn't figure out how to resolve these otherwise.

Reply:
Ah cool thanks. Done